DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: rtables.officer
 Title: Reporting Tables
```

Comment:
This is the same as `rtables`. Is it an issue ?

Reply:
This was also carried over from `rtables` in the README: https://github.com/insightsengineering/rtables.officer/blob/9afc5a8b67bb1d23dda3a38e771360c3540aba05/README.Rmd#L42

@shajoezhu is there an alternative title you'd like to use to describe the package?